### DISCUSSION TITLE
[Feature Request] Twilight Forest Door item models fix

### DISCUSSION THREAD

**Discussion by Vapaman** [ACGaming/UniversalTweaks#320] [Bugfix Suggestions]
Twilight Forests doors inventory textures have been broken due to missing item models for multiple versions. It was finally fixed in more modern versions. Bringing this fix to 1.12.2 would soothe many 1.12 players minds and eyes.

![kuva](https://github.com/ACGaming/UniversalTweaks/assets/77460466/73917f08-a8ea-457c-b418-a9e86fd70e5c)

**Comment by magcicada** [comment]
![image](https://github.com/ACGaming/UniversalTweaks/assets/45464670/d5eb17ca-e987-42e9-8c04-9087503889bf)
